### PR TITLE
Create a basic pipeline for GTFS CSVs

### LINKED ISSUE #48: Create a basic pipeline for GTFS CSVs
GTFS is a transportation format, see details here: https://www.gov.il/he/Departments/General/gtfs_general_transit_feed_specifications
The data is here: ftp://gtfs.mot.gov.il

This task is to create a basic pipeline that downloads the files and for now, just prints the number of lines in each file (we will later replace this with something more interesting).

You can see an example pipeline here:
https://github.com/hasadna/hasadna/tree/master/projects/data_analysis
To run the pipeline, follow the instructions in that link.

Note this task is similar task to https://github.com/hasadna/hasadna/issues/47
The difference is, that the data is in GTFS format.

So let's break this up a little bit:
1. Run the example pipeline using `bazel run`
2. Explore the example pipeline, go over the files and try to understand who's calling who.
3. Copy it into `projects/opentrain/gtfs_pipeline`, make it work there as is.
4. Rename to `gtfs_pipeline`, make sure it still works :)
5. Replace the data it uses with the CSVs above. The example pipeline's data is defined here:
https://github.com/hasadna/hasadna/blob/master/WORKSPACE#L145-L167

If you encounter any issue, please ask!

### PR DESCRIPTION
**Pull request by yehudalev**
Fixes #48

### CODE REVIEW

**Comment by yehudalev** [comment]
can we assume what are the names of the files?

File: projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+class DataHandler {
+      public static void main(String[] args) throws IOException {
+        System.out.println(args[0]);
+        String outPut = "";
+        Path outfilePath = Paths.get(args[args.length-1]);
+        for (String path : args) {
+            Path curPath = Paths.get(path);
+            List<String> curPathContent = Files.readAllLines(curPath);
+            outPut = outPut + curPath.getFileName() + " lines: ";
```
Comment (oferb) [inline projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java]: You can do this instead:
`output += curPath.getFileName() + " lines: ";`

File: projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            outPut = outPut + String.valueOf(curPathContent.size()) + "\n";
```
Comment (oferb) [inline projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java]: Java will automatically convert `curPathContent.size()` to String, you don't need `String.valueOf()`.

File: projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            Files.write(outfilePath, new String(curPath.getFileName()+"\n").getBytes(), StandardOpenOption.APPEND, StandardOpenOption.CREATE);
+            Files.write(outfilePath, curPathContent, StandardOpenOption.APPEND);
+            Files.write(outfilePath, new String("\n").getBytes(), StandardOpenOption.APPEND);
+
+        }
+        System.out.print(outPut);
+    }
+}
```
Comment (oferb) [inline projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java]: Please end files in a newline, so they don't have this red mark.
It's standard in text files.

File: projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.List;
+import java.lang.String;
+
+class DataHandler {
+      public static void main(String[] args) throws IOException {
+        System.out.println(args[0]);
+        String outPut = "";
```
[the remark this answers is not in the document]
Comment (yehudalev) [reply projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java]: You're right, I used this statement in the first place for debugging and I forgot to delete it. I will fix it

File: projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.List;
+import java.lang.String;
+
+class DataHandler {
+      public static void main(String[] args) throws IOException {
+        System.out.println(args[0]);
```
[the remark this answers is not in the document]
Comment (yehudalev) [reply projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java]: You're right, I used this statement in the first place for debugging and I forgot to delete it. I will fix it

File: projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        String outPut = "";
+        Path outfilePath = Paths.get(args[args.length-1]);
```
[the remark this answers is not in the document]
Comment (yehudalev) [reply projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java]: Fixed

File: projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java (hunk 1 above)
Comment (yehudalev) [reply projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java]: Fixed

File: projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java (hunk 2 above)
Comment (yehudalev) [reply projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java]: you're right, fixed

File: WORKSPACE [hunk 7]
```diff
@@ -39,8 +39,8 @@ android_sdk_repository(
 # MARK: StartupOS start
 http_archive(
     name = "startup_os",
-    urls = ["https://github.com/google/startup-os/archive/b07efbc4bee807026641eba63f4c36e804e8233e.zip"],
-    strip_prefix = "startup-os-b07efbc4bee807026641eba63f4c36e804e8233e"
+    urls = ["https://github.com/google/startup-os/archive/672fba099b81a45ddd9c28a39ea783d0435f2b00.zip"],
```
Comment (val-fed) [inline WORKSPACE]: s / 672fba099b81a45ddd9c28a39ea783d0435f2b00 / b07efbc4bee807026641eba63f4c36e804e8233e

File: WORKSPACE [hunk 8]
```diff
@@ -39,8 +39,8 @@ android_sdk_repository(
...
+    strip_prefix = "startup-os-672fba099b81a45ddd9c28a39ea783d0435f2b00"
```
Comment (val-fed) [inline WORKSPACE]: s / 672fba099b81a45ddd9c28a39ea783d0435f2b00 / b07efbc4bee807026641eba63f4c36e804e8233e

**Pull request merged**